otel/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/otel_lib/tracer.py

```diff
@@ -421,12 +421,12 @@ def _persist_run(self, run: Run) -> None:
         try:
             self._convert_run_to_spans(run.dict(), modelmetrics)
             for modelmetric in modelmetrics:
-                token_counter = self.meter.create_up_down_counter(f"{modelmetric.llm}_{modelmetric.model}_token")
-                call_counter = self.meter.create_up_down_counter(f"{modelmetric.llm}_{modelmetric.model}_calls")
-                duration_counter = self.meter.create_up_down_counter(f"{modelmetric.llm}_{modelmetric.model}_duration")
-                token_counter.add(modelmetric.token)
-                call_counter.add(modelmetric.calls)
-                duration_counter.add(modelmetric.duration)
+                token_counter = self.meter.create_up_down_counter(f"llm.request.token")
+                call_counter = self.meter.create_up_down_counter(f"llm.request.count")
+                duration_counter = self.meter.create_up_down_counter(f"llm.response.duration.avg")
+                token_counter.add(modelmetric.token, {"model_id": modelmetric.model, "user_id": "user1"})
```

Comment:
we also want to identify the llm type/vendor info, so for the metrics labels, could you add the `modelmetric.llm` as llm_name?

Reply:
Updated here. But we are supposed to follow the definition of the Semantic Convention. After the Convention is determined, all metrics definitions in Agent and DC will be changed accordingly.